subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -163,6 +173,12 @@ static bool validate_firmware(u32_t fw_dst_address, u32_t fw_src_address,
 					(u8_t *)fw_val_info->firmware_address,
 					fwinfo->firmware_size);
 
+		if ((retval == 0)/* && !external*/) {
```

Comment:
I guess since this a draft this is not complete?

Reply:
I guess ignore this comment I went straight in with reading the bold text.

Reply:
np :). It's commented out because it's waiting for another PR where `external` is introduced.